modeshape-parent/pom.xml

```diff
@@ -181,6 +181,7 @@
         <version.org.jboss.ironjacamar>1.0.13.Final</version.org.jboss.ironjacamar>
         <version.com.datastax.cassandra>2.0.0-rc2</version.com.datastax.cassandra>
         <version.org.apache.cassandra>2.0.3</version.org.apache.cassandra>
+        <version.org.apache.pdfbox>1.8.12</version.org.apache.pdfbox>
```

Comment:
can you please change this to be `1.8.10`.  ModeShape already uses Apache Tika for text and mime-type extraction which in turn uses this version:
```
 org.apache.tika:tika-parsers:jar:1.12:compile
 +- org.gagravarr:vorbis-java-tika:jar:0.6:compile
 +- org.apache.james:apache-mime4j-core:jar:0.7.2:compile
 +- org.apache.james:apache-mime4j-dom:jar:0.7.2:compile
 +- org.apache.commons:commons-compress:jar:1.10:compile
 +- org.tukaani:xz:jar:1.5:compile
 +- commons-codec:commons-codec:jar:1.9:compile
 +- org.apache.pdfbox:pdfbox:jar:1.8.10:compile
 |  +- org.apache.pdfbox:fontbox:jar:1.8.10:compile
 |  +- org.apache.pdfbox:jempbox:jar:1.8.10:compile
```